src/Faucet.sol

```diff
@@ -51,9 +51,8 @@ contract Faucet is Initializable, UUPSUpgradeable, OwnableUpgradeable, IFaucet {
      * @dev Upgrade calling `upgradeTo()`
      */
     function initialize() external initializer {
-        __Ownable_init();
+        __Ownable_init(msg.sender);
```

Comment:
`Ownable` now must receive the owner as param (more info [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md?plain=1#L69))